src/mono/netcore/sample/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <PropertyGroup>
+    <!-- Samples don't always follow good programming style -->
+    <RunAnalyzers>false</RunAnalyzers>
```

Comment:
I tried :) Ok.

Reply:
> I tried :) Ok.

@stephentoub turns out the samples are just _too good_ as a debugging aid for the runtime team.  And we copy/paste all kinds of problematic code in there. 😁